### ISSUE TITLE
regression: premission denied while building using w64devkit

### ISSUE THREAD

**Issue by avih** [rmyorston/busybox-w32#284]
When replacing `busybox.exe` at w64devkit/bin with a newly built version, e.g. the currently-last prerelease busybox-w64-PRE-4854-ge29a10645.exe, and trying to build busybox-w32 master using w64devkit (while setting `BB_OVERRIDE_APPLETS="ar make"`, there are permission errors like this, typically during `make mingw64_defconfig` (but with some versions also at the later `make` command):
```
: scripts/basic/fixdep: Permission denied
: scripts/basic/fixdep: Permission denied
make[1]: *** [scripts/Makefile.build:197: archival/bbunzip.o] Error 126
make[1]: *** Waiting for unfinished jobs....
make[1]: *** [scripts/Makefile.build:197: archival/cpio.o] Error 126
```

As far as I can tell, the first version of the pre-releases which has the issue is busybox-w64-PRE-4800-g67aec6403.exe (2022-12-22)

The latest version which works and builds busybox-w32 successfully is the currently-last release - busybox-w64-FRP-4784-g5507c8744.exe  (2022-11-09)

**Comment by rmyorston** [comment]
Without even reading the full problem description I knew this was going to be commit b11352dcb. It's proved rather troublesome.

Should be fixed in PRE-4855.

**Comment by avih** [comment]
> Should be fixed in PRE-4855.

Thanks, confirmed fixed.

FWIW, if you're already building using w64devkit, then I think it would be useful dogfooding to use a recent busybox.exe instead of the one which ships with w64devkit. This regression would have probably come up much sooner had that been the case.

As far as I can tell, at this time, the differences from upstream busybox-w32 are few minor line edit related patches, and the following applets disabled:
https://github.com/skeeto/w64devkit/blob/43707da51f7b76a0ead3bd5a62222e8caecb38d9/Dockerfile#L357-L370

For which I think it's enough to use standard upstream busybox-w32 binary, and disable only those: `BB_OVERRIDE_APPLETS="ar make strings"` in order to use it to build busybox-w32 (I didn't override `strings` before, and it still worked, but w64devkit does ship its own `strings.exe`).

**Comment by rmyorston** [comment]
My development workflow is Linux-based.

I'll add a check with w64devkit to my release procedure, though. It seems like a useful stress test.

**Comment by avih** [comment]
> My development workflow is Linux-based.

Right, I had the impression the official and pre-release builds are now built using w64devkit, but now I think I misinterpreted some statement. Thanks.
